### PR TITLE
Create ur.json

### PR DESCRIPTION
**Pull request by faisalfh5**
We are native Urdu speakers and we would like to work on the Urdu translation of the solidity docs. We are co-workers from [Falcon IT Consulting] (https://github.com/falconitconsultant) and We are using Solidity on a daily basis.
As a team lead of more than 20 developers, we’ve localized a lot of Pakistani websites including the most famous [Wifaq Ul Madaris] (https://www.wifaqulmadaris.org/).
So I have enough experience to maintain an open-source project.
Members: yousufkalim, faisalfh5, Jawad0318

### CODE REVIEW

**Comment by faisalfh5** [comment]
Urdu translation of the Solidity docs

**Review by r0qs — APPROVED** [review]
Thanks @faisalfh5 :)

We will add the `ur-urdu` repo as soon as this PR is merged.

**Review by cameel — APPROVED** [review]

**Review by r0qs — APPROVED** [review]

**Comment by r0qs** [comment]
@faisalfh5 the repository was created here https://github.com/solidity-docs/ur-urdu
Please, follow the instructions at our maintainer docs: https://github.com/solidity-docs/.github/blob/main/maintainer-guide.md#starting-a-new-language to setup the repo.

If you have any issues, please reach out us through our [matrix channel](https://matrix.to/#/#ethereum_solidity-dev:gitter.im).

**Pull request merged**
